xgboost_ames_housing/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
+  - gaocegege
```

Comment:
Why gaocegege, jlewi, lunkai?

I would suggest just listing you and richard for now since you are the only two people actively working on this example.

We can others in the future if needed be.

Reply:
makes sense. changed it.